Index: once_cell-1.16.0/Cargo.toml
===================================================================
--- once_cell-1.16.0.orig/Cargo.toml
+++ once_cell-1.16.0/Cargo.toml
@@ -66,7 +66,7 @@ required-features = ["std"]
 name = "test_synchronization"
 required-features = ["std"]
 
-[dependencies.atomic_polyfill]
+[dependencies.atomic_polyfill_renamed]
 version = "1"
 optional = true
 package = "atomic-polyfill"
@@ -100,7 +100,7 @@ alloc = ["race"]
 atomic-polyfill = ["critical-section"]
 critical-section = [
     "critical_section",
-    "atomic_polyfill",
+    "atomic_polyfill_renamed",
 ]
 default = ["std"]
 parking_lot = ["parking_lot_core"]
